lib/models/Flower.js

```diff
@@ -0,0 +1,91 @@
+const pool = require('../utils/pool');
+
+module.exports = class Flower {
+  id;
+  name;
+  color;
+
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.color = row.color;
+  }
+
+  static async insert({ name, color }) {
+    const { rows } = await pool.query(
+      `INSERT INTO
+        flowers (name, color)
+      VALUES
+        ($1, $2)
+      RETURNING
+        *
+      `,
+      [name, color]
+    );
```

Comment:
We might add early returns here for all of static methods in case the case of nothing being returned from the query we can avoid some errors.